Data/ByteString/Lazy.hs

```diff
@@ -41,6 +41,11 @@
 --
 -- > import qualified Data.ByteString.Lazy as B
 --
+-- Note: Using lazy I\/O functions like 'readFile' or 'hGetContents' largely
+-- gives up on prompt resource management. The underlying files or sockets
```

Comment:
"prompt resource management" seems rather vague. I think it would be better to talk concretely about possible risks.